build.gradle

```diff
@@ -308,16 +311,17 @@ dependencies {
     implementation group: 'org.springframework', name: 'spring-webmvc', version: versions.springVersion
 
     implementation group: 'com.github.ben-manes.caffeine', name: 'caffeine', version: '3.0.4'
-    implementation (group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.13.2.1') {
-        force = true
-    }
-    implementation (group: 'com.fasterxml.jackson.core', name: 'jackson-core', version: versions.jackson) {
-        force = true
-    }
+    implementation (group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.15.2')
```

Comment:
can't we use 2.13.2 ( version: versions.jackson))

Reply:
local dependency check scan flags jackson-databind 2.13.2 as vulnerable to CVE-2020-36518, CVE-2022-42003, CVE-2022-42004